ldap-login-admin-form.php

```diff
@@ -140,7 +140,7 @@ function admin_form(&$qa_content) {
 
         array(
           'id' => 'ldap_login_ad_pwd_display',
-          'label' => 'Password for AD binging accout',
+          'label' => 'Password for AD binding accout',
```

Comment:
Well you missed the `accout` ;)

```suggestion
          'label' => 'Password for AD binding account',
```